course/models.py

```diff
@@ -453,15 +559,25 @@ class FlowPageVisit(models.Model):
     # This is redundant (because the FlowSession is available through
     # page_data), but it helps the admin site understand the link
     # and provide editing.
-    flow_session = models.ForeignKey(FlowSession, db_index=True)
-
-    page_data = models.ForeignKey(FlowPageData, db_index=True)
-    visit_time = models.DateTimeField(default=now, db_index=True)
-    remote_address = models.GenericIPAddressField(null=True, blank=True)
-
-    is_synthetic = models.BooleanField(default=False)
-
-    answer = JSONField(null=True, blank=True)
+    flow_session = models.ForeignKey(FlowSession, db_index=True,
+            verbose_name = 'Flow session')
+
+    page_data = models.ForeignKey(FlowPageData, db_index=True,
+            verbose_name = 'Page data')
+    visit_time = models.DateTimeField(default=now, db_index=True,
+            verbose_name = 'Visit time')
+    remote_address = models.GenericIPAddressField(null=True, blank=True,
+            verbose_name = 'Remote address')
+
+    is_synthetic = models.BooleanField(default=False,
+            help_text="NEED HELP TEXT",
```

Comment:
help_text="Synthetic flow page visits are generated for unvisited pages once a flow is finished. This is needed since grade information is attached to a visit, and it needs a place to go."